test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -435,11 +435,11 @@ describe('CSOT spec prose tests', function () {
        *    - Expect this to fail with a timeout error.
        * 1. Verify that an `aggregate` command and two `getMore` commands were executed against the `db.coll` collection during the test.
        */
-      it.skip('sends correct number of aggregate and getMores', metadata, async function () {
+      it('sends correct number of aggregate and getMores', metadata, async function () {
         const changeStream = client
           .db('db')
           .collection('coll')
-          .watch([], { timeoutMS: 20, maxAwaitTimeMS: 19 });
+          .watch([], { timeoutMS: 120, maxAwaitTimeMS: 10 });
         const maybeError = await changeStream.next().then(
           () => null,
           e => e
```

Comment:
Can we leave a comment about why we don't check for "return a change stream with a non-zero ID.".